docs/react/guides/ssr.md

```diff
@@ -151,371 +153,376 @@ function Posts() {
 
 The setup is minimal and this can be a quick solution for some cases, but there are a **few tradeoffs to consider** when compared to the full approach:
 
-- If you are calling `useQuery` with the same query in multiple locations, you need to pass `initialData` to all of them
+- If you are calling `useQuery` in a component deeper down in the tree you need to pass the `initialData` down to that point
+- If you are calling `useQuery` with the same query in multiple locations, passing `initialData` to only one of them can be brittle and break when your app changes since. If you remove or move the component that has the `useQuery` with `initialData`, the more deeply nested `useQuery` might no longer have any data. Passing `initialData` to **all** queries that needs it can also be cumbersome.
 - There is no way to know at what time the query was fetched on the server, so `dataUpdatedAt` and determining if the query needs refetching is based on when the page loaded instead
+- If there is already data in the cache for a query, `initialData` will never overwrite this data, even if the new data is fresher than the old one
```

Comment:
I would emphasize here that this happens on every client side transition. In the `getServerSideProps` example, if you navigate a way from the page and then back again, you would:

- fetch `posts` again on the server
- send them down to the client as `initialData`
- but there, it would be completely ignored.

I think this is the biggest downside of the `initialData` approach

Reply:
Good point! I'll make this clearer.